src/LinAlg/hiopLinSolverSparseGinkgo.cpp

```diff
@@ -232,28 +232,46 @@ std::shared_ptr<gko::Executor> create_exec(std::string executor_string)
 }
 
 
+gko::solver::trisolve_algorithm create_alg(std::string algorithm_string)
+{
+    std::map<std::string, gko::solver::trisolve_algorithm>
```

Comment:
wouldn't be more clear to have `alg_map` a static member in the class? 

Also the indentation is a bit weird. Maybe something like this?
```
alg_map{{"syncfree", gko::solver::trisolve_algorithm::syncfree},
        {"sparselib", gko::solver::trisolve_algorithm::sparselib}};
```